### PR TITLE
[HAMMER] Make builds work

### PR DESCRIPTION
**Pull request by carbonin**
These changes have been made on the appliance side, but have been missing here for months.

Specifically, this PR updates to node 8, creates the `vmdb/log` directory when trying to create the `apache` log directory, and updates to bundler `>=1.16.2`

### CODE REVIEW

File: images/miq-app/Dockerfile [hunk 1]
```diff
@@ -82,14 +82,14 @@ WORKDIR ${APP_ROOT}
 RUN source /etc/default/evm && \
     export RAILS_USE_MEMORY_STORE="true" && \
     npm install bower yarn -g && \
-    gem install bundler -v 1.15.4 && \
+    gem install bundler -v ">=1.16.2" && \
```
Comment (simaishi) [inline images/miq-app/Dockerfile]: Not a big deal.. but where did this `1.16.2` come from? In appliance, we have `gem install bundler -v ">=1.8.4"`

Comment (carbonin) [reply images/miq-app/Dockerfile]: Well 1.15 was causing a failure to resolve so I figured it should be updated. @bdunne also mentioned that 1.16.1 had some problem, but I can't remember what it was.

The appliance should probably be updated too.

File: images/miq-app/Dockerfile [hunk 2]
```diff
@@ -48,7 +49,6 @@ RUN yum -y install centos-release-scl-rh && \
                    net-tools               \
                    nmap-ncat               \
                    nodejs                  \
-                   npm                     \
```
[the remark this answers is not in the document]
Comment (carbonin) [reply images/miq-app/Dockerfile]: Yes

File: images/miq-app/Dockerfile [hunk 3]
```diff
@@ -30,7 +30,8 @@ RUN curl -sSLko /etc/yum.repos.d/manageiq-ManageIQ-Master-epel-7.repo \
       https://copr.fedorainfracloud.org/coprs/manageiq/ManageIQ-Master/repo/epel-7/manageiq-ManageIQ-Master-epel-7.repo
 
 ## Install EPEL repo, yum necessary packages for the build without docs, clean all caches
-RUN yum -y install centos-release-scl-rh && \
+RUN yum -y install centos-release-scl-rh \
+                   https://rpm.nodesource.com/pub_8.x/el/7/x86_64/nodesource-release-el7-1.noarch.rpm && \
```
[the remark this answers is not in the document]
Comment (carbonin) [reply images/miq-app/Dockerfile]: Nope, I'm doing the same thing we do [in the appliance](https://github.com/ManageIQ/manageiq-appliance-build/blob/5dbfc1c558244aab0ef0659f1ccaf390145d370f/kickstarts/partials/main/repos.ks.erb#L9)

File: images/miq-app/Dockerfile (hunk 1 above)
Comment (carbonin) [reply images/miq-app/Dockerfile]: Okay, but the ManageIQ appliance is at `1.16.5`

Comment (simaishi) [reply images/miq-app/Dockerfile]: It's not a problem to use 1.16 upstream as I don't think we really depends on bundler version for our functionality and this is how it has always been in master (now hammer).  But if we require `1.16.2` to be minimum (in Gemfile), that will break the downstream build...

Comment (carbonin) [reply images/miq-app/Dockerfile]: Ah okay, I didn't realize that was a response to @bdunne 's comment rather than the change on this line.

Comment (simaishi) [reply images/miq-app/Dockerfile]: oh sorry, that comment wasn't clear...

**Pull request merged**